app/points/domain/default-contracted-hours.js

```diff
@@ -0,0 +1,15 @@
+const assertNumber = require('./validation/assert-number')
+
+class DefaultContractedHours {
+  constructor (psoHours, otherHours) {
+    this.pso = psoHours
+    this.other = otherHours
```

Comment:
The `other` in this scenario can be safely categorised as `po` as seen here: https://drive.google.com/drive/folders/0Byh_XMVkeN0EOWowbmN6bmRucnc?ths=true